package.json

```diff
@@ -9,31 +9,34 @@
     ]
   },
   "scripts": {
-    "account": "yarn workspace @se-2/hardhat account",
-    "chain": "yarn workspace @se-2/hardhat chain",
-    "fork": "yarn workspace @se-2/hardhat fork",
-    "deploy": "yarn workspace @se-2/hardhat deploy",
-    "verify": "yarn workspace @se-2/hardhat verify",
-    "compile": "yarn workspace @se-2/hardhat compile",
-    "generate": "yarn workspace @se-2/hardhat generate",
-    "hardhat:lint": "yarn workspace @se-2/hardhat lint",
-    "hardhat:lint-staged": "yarn workspace @se-2/hardhat lint-staged",
-    "hardhat:test": "yarn workspace @se-2/hardhat test",
-    "start": "yarn workspace @se-2/nextjs dev",
-    "next:lint": "yarn workspace @se-2/nextjs lint",
-    "next:format": "yarn workspace @se-2/nextjs format",
-    "next:check-types": "yarn workspace @se-2/nextjs check-types",
+    "account": "pnpm --filter @se-2/hardhat account",
+    "chain": "pnpm --filter @se-2/hardhat chain",
+    "fork": "pnpm --filter @se-2/hardhat fork",
+    "hardhat:deploy": "pnpm --filter @se-2/hardhat hardhat:deploy",
+    "verify": "pnpm --filter @se-2/hardhat verify",
+    "compile": "pnpm --filter @se-2/hardhat compile",
+    "generate": "pnpm --filter @se-2/hardhat generate",
+    "hardhat:lint": "pnpm --filter @se-2/hardhat lint",
+    "hardhat:lint-staged": "pnpm --filter @se-2/hardhat lint-staged",
+    "hardhat:test": "pnpm --filter @se-2/hardhat test",
+    "start": "pnpm --filter @se-2/nextjs dev",
+    "next:lint": "pnpm --filter @se-2/nextjs lint",
+    "next:format": "pnpm --filter @se-2/nextjs format",
+    "next:check-types": "pnpm --filter @se-2/nextjs check-types",
     "postinstall": "husky install",
     "precommit": "lint-staged",
-    "vercel": "yarn workspace @se-2/nextjs vercel",
-    "vercel:yolo": "yarn workspace @se-2/nextjs vercel:yolo"
+    "vercel": "pnpm --filter @se-2/nextjs vercel",
+    "vercel:yolo": "pnpm --filter @se-2/nextjs vercel:yolo"
   },
-  "packageManager": "yarn@3.2.3",
+  "packageManager": "pnpm@8.3.1",
```

Comment:
why 8.3.1, not 8.6.10?

Reply:
Got this when running `pnpm install`

![image](https://github.com/scaffold-eth/scaffold-eth-2/assets/2486142/0929a306-9463-40b0-b2c9-81884bd21898)

Reply:
Updated at [d148eb0](https://github.com/scaffold-eth/scaffold-eth-2/pull/444/commits/d148eb050c6ac96892db959b700fda9b6a8d452e)